src/platforms/common/performance/capturing/automatic.mdx

```diff
@@ -12,38 +13,251 @@ To _automatically_ capture transactions, you must first <PlatformLink to="/perfo
 
 </Note>
 
-The `@sentry/tracing` package provides a `BrowserTracing` integration to add _automatic_ instrumentation for monitoring the performance of browser applications.
+The `@sentry/react-native` provides a `ReactNativeTracing` integration to add _automatic_ instrumentation for monitoring the performance of React Native applications.
```

Comment:
This is now changing the common/shared content though. You probably want to have an override then specific for react native

Reply:
Yes, so this is just me writing everything out in one page ignoring the shared content. Will later split everything up to work with shared content.